### PR TITLE
correct schema error

### PR DESCRIPTION
**Pull request by wandmagic**
# Committer Notes

### All Submissions:

- [ ] Have you followed the guidelines in our [Contributing](https://github.com/usnistgov/metaschema-xslt/blob/master/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/metaschema-xslt/pulls) for the same update/change?
- [ ] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [ ] Do all automated CI/CD checks pass?

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you included examples of how to use your new feature(s)?
- [ ] Have you updated all [website](https://pages.nist.gov/metaschema) and readme documentation affected by the changes you made? Changes to the website can be made in the website/content directory of your branch.

### CODE REVIEW

File: src/compose/testing/models_metaschema.xml [hunk 1]
```diff
@@ -36,6 +36,9 @@
         <flag ref="id"/>
         <model>
             <field ref="field-1only"/>
+            <field>
+                <use-name>overloaded</use-name>
+            </field>
```
Comment (aj-stein-nist) [inline src/compose/testing/models_metaschema.xml]: Can you explain a little more in the comment or issue what this should be accomplishing? A `field` "by ref" that is `@ref`less should not be allowed and should not work, but I guess I would need to know more context to understand why this change is valid.

**Review by aj-stein-nist — CHANGES_REQUESTED** [review]

File: src/compose/testing/models_metaschema.xml (hunk 1 above)
Comment (wandmagic) [reply src/compose/testing/models_metaschema.xml]: when the JSON Schema is generated, the ref that is specified in the other overloaded fields doesn't exist causing an error. this field is so that the ref parameters in the other fields actually point to something

Comment (wandmagic) [reply src/compose/testing/models_metaschema.xml]: I've removed that part and we can fix it with the other PR with the json schema testing
